src/displayapp/fonts/fonts.json

```diff
@@ -0,0 +1,78 @@
+{
+   "jetbrains_mono_bold_20": {
+      "sources": [
+         {
+            "file": "JetBrainsMono-Bold.ttf",
+            "range": "0x20-0x7e, 0x410-0x44f"
+         },
+         {
+            "file": "FontAwesome5-Solid+Brands+Regular.woff",
+            "range": "0xf293, 0xf294, 0xf244, 0xf240, 0xf242, 0xf243, 0xf241, 0xf54b, 0xf21e, 0xf1e6, 0xf54b, 0xf017, 0xf129, 0xf03a, 0xf185, 0xf560, 0xf001, 0xf3fd, 0xf069, 0xf1fc, 0xf45d, 0xf59f, 0xf5a0, 0xf029, 0xf027, 0xf028, 0xf6a9, 0xf04b, 0xf04c, 0xf048, 0xf051, 0xf095, 0xf3dd, 0xf04d, 0xf2f2, 0xf024, 0xf252, 0xf569, 0xf201, 0xf06e, 0xf015"
+         }
+      ],
+      "bpp": 1,
+      "size": 20,
+      "patches": [["patch", "{file}", "{file}_zero.patch"]]
```

Comment:
I think supplying just the name of the patch file should be enough here. It doesn't need to be able to run any command.

Reply:
done. (I left in that if the script fails using a simple `patch`, it will try using as a whole command)

Reply:
I don't think the `{file}` overriding is useful for filenames here, and I'm not sure falling back to executing as a command is necessary. I'd rather remove features that aren't necessary since they won't get maintained.

Reply:
Good question. As far as I can tell, that capability just requires one extra for loop, so it's not that bad. If a font required two changes, my immediate thought would be that they should be two separate patches, but I'm not 100% sure.

Reply:
then I'll leave that in.